include/zephyr/drivers/regulator.h

```diff
@@ -567,6 +567,21 @@ static inline int regulator_get_error_flags(const struct device *dev,
 	return api->get_error_flags(dev, flags);
 }
 
+/**
+ * @brief Get minimal voltage from common config.
+ *
+ * @param dev Regulator device instance.
+ *
+ * @retval minimal supported voltage from common config.
+ */
+static inline int32_t regulator_get_cfg_min_uv(const struct device *dev)
+{
+	const struct regulator_common_config *cmn =
+		(const struct regulator_common_config *)dev->config;
+
+	return cmn->min_uv;
+}
+
```

Comment:
this is an internal API, meant for drivers only. Please name it `regular_common_get_min_voltage()` and move it to the internal section where common functions are present.